scripts/staking.py

```diff
@@ -75,7 +75,7 @@
         if args.unstake:
             if not is_service_staked(ledger_api, args.service_id, args.staking_contract_address):
                 # the service is not staked, so we don't need to do anything
-                print(f"Service {args.service_id} is not staked. Exiting...")
+                print(f"Service {args.service_id} is not staked.")
```

Comment:
Why removed?

Reply:
Because this exit point does not finish the `run_service.sh` (or other) script: it exits with an output code of 0.

 "Exiting..." here was a bit misleading, because the user could interpret that it was meant to exit the full script, when it was just exitting the staking command within the script.